mifospay/src/main/res/layout/activity_mobile_verification.xml

```diff
@@ -120,17 +123,21 @@
             android:layout_marginTop="@dimen/value_10dp"
             android:orientation="horizontal">
 
-            <EditText
-                android:id="@+id/et_otp"
+            <android.support.design.widget.TextInputLayout
+                style="@style/Widget.MaterialComponents.TextInputLayout.OutlinedBox"
                 android:layout_width="wrap_content"
-                android:layout_height="48dp"
-                android:background="@drawable/bg_et_round_border"
-                android:hint="@string/enter_otp"
-                android:inputType="number"
-                android:paddingLeft="15dp"
-                android:paddingRight="15dp"
-                android:visibility="gone"
-                android:maxLength="6"/>
+                android:layout_height="@dimen/value_48dp"
+                android:hint="Enter OTP" >
```

Comment:
As stated in the previous review, please refrain hardcoding strings and values

Reply:
Can you please tell me where not to hardcode?Because I thought only the height and width is not supposed to be hard coded.Is the padding and alignment features not supposed to be hardcoded